ecf/defs/prod00.def

```diff
@@ -9,24 +9,29 @@ suite prod00
   edit ECF_TRIES '1'
   edit CYC '00'
   edit ENVIR 'prod'
-  edit PROJ 'OPS'
-  edit E 'j'
-  edit QUEUE 'prod'
+  edit PROJ 'GFS'
+  edit PROJENVIR 'DEV'
+  edit E ''
+  edit QUEUE 'dev'
   edit QUEUESHARED 'dev_shared'
   edit QUEUESERV 'dev_transfer'
-  edit PROJENVIR 'OPS'
-  edit MACHINE_SITE 'production'
+#### Developer overwrite
+  edit ECF_INCLUDE '/lfs/h2/emc/global/noscrub/Lin.Gan/git/feature-ops-wcoss2/ecf/include'
+  edit MACHINE_SITE 'development'
+####
 
   family gfs
     edit CYC '00'
-    edit ECF_FILES '/ecf/ecfnets/scripts/gfs_FV3'
-    edit PROJ 'GFS-OPS'
+    edit PACKAGEHOME '/lfs/h2/emc/global/noscrub/Lin.Gan/git/feature-ops-wcoss2'
```

Comment:
This variable definition should be outside the `gfs` family as it is a constant.  Same applies for this variable defined on line 2257.

Reply:
Steven said in an email the PACKAGEHOME should to be inside family level.

Reply:
His team will move all common variables from family to suite level. That includes this variable and the other QUEUE variables.